.github/workflows/qa.yaml

```diff
@@ -91,9 +91,15 @@ jobs:
           deb http://ddebs.ubuntu.com $(lsb_release -cs)-updates main restricted universe multiverse
           deb http://ddebs.ubuntu.com $(lsb_release -cs)-proposed main restricted universe multiverse" | \
           sudo tee -a /etc/apt/sources.list.d/ddebs.list
-          # Sometimes ddebs archive is stuck, so in case of failure we ignore this step, it's not fundamental.
+          # Sometimes ddebs archive is stuck, so in case of failure we need to go manual
           sudo apt update -y || true
```

Comment:
While you are at it, remove this `|| true` as well :wink:

Reply:
I was thinking about it, but I decided not to do it, because IIRC in case we've troubles with `ddebs` archives `apt update` would fail (since there may be signatures issues), so it's just safer not to do it. Since at the end what we care is about not failing on installing the ddebs files. And that's not handled.